README.md

```diff
@@ -11,47 +11,116 @@ ADaM in R Asset Library
 
 ## Purpose
 
-To provide an open source, modularized toolbox that enables the pharmaceutical programming community to develop ADaM datasets in R.
+To provide an open source, modularized toolbox that enables the pharmaceutical programming community
+to develop ADaM datasets in R.
 
 ## Installation
 
-Once the package is available from CRAN you'll be able to install it using `install.packages("admiral")`.
+The package is available from CRAN and can be installed by running `install.packages("admiral")`.
 
-In the meantime you can install the latest release of the package directly from GitHub.
+To install the latest development version of the package directly from GitHub use the following code:
 
 ```r
 if (!requireNamespace("remotes", quietly = TRUE)) {
   install.packages("remotes")
 }
-remotes::install_github("pharmaverse/admiral.test", ref = "main") # This is a required dependency of {admiral}
-remotes::install_github("pharmaverse/admiral", ref = "main")
+remotes::install_github("pharmaverse/admiral", ref = "devel")
 ```
 
-## Scope
-
-* A toolbox of re-usable functions and utilities to create ADaM datasets using R scripts in a modular manner (an "opinionated" design strategy)
-* Pharmaceutical communities and companies are encouraged to contribute to `{admiral}` following the provided programming strategy and modular approach
-* All functions are documented, tested, include examples and are listed in the
-[Reference section](https://pharmaverse.github.io/admiral/reference/index.html)
-* Vignettes on how to create ADSL, BDS and OCCUR datasets, including example scripts
-* Vignettes for ADaM dataset specific functionality (i.e. dictionary coding, date imputation, SMQs ...)
+## Main Goal
 
-## Usage
+Provide data scientists with an open source, modularized toolbox with which to create ADaM datasets
+in R from. _As opposed to a “run 1 line and an ADaM appears” black-box solution or an attempt to
+automate ADaM._
 
-* Think of `{admiral}` as a toolbox of modular blocks (R functions) to create analysis derivations
-    * Each block has a stand alone purpose, i.e. each function provides a specific functionality
-    * Data Scientists can create their own blocks, i.e. create own R functions
-* Constructing ADaM dataset should become like building out of blocks that are based on `{admiral}` modular functions and user created modular functions
+One of the key aspects of `{admiral}` is its development by the users for the users.
+It gives an entry point for all data scientists to collaborate, co-create and contribute to a
```

Comment:
Feel like our audience is statistical programmers, but I guess the data scientists are taking over the world!  Could we make it less specific to a job role?